keras/losses.py

```diff
@@ -41,6 +56,13 @@ def categorical_hinge(y_true, y_pred):
 
 
 def logcosh(y_true, y_pred):
+    """Logarithm of the hyperbolic cosine of the prediction error
+
+    `log(cosh(x))` is approximately equal to `x**2 / 2` for small `x` and
+    to `abs(x)` for large `x`. This means it works mostly like the mean
+    squared error, but will not be so strongly affected by the occasional
+    wildly incorrect prediction.
```

Comment:
This one is reasonable. Style fixes required: the one-line summary should end with a dot. The code snippets should follow PEP8 (spaces around operators like `**`). Docstrings should have a `# Arguments` and `# Returns` section.